packages/fx-core/src/component/debugHandler/appManifest.ts

```diff
@@ -153,6 +153,9 @@ export class AppManifestDebugHandler {
       this.envInfoV3.state[ComponentNames.AppManifest] =
         this.envInfoV3.state[ComponentNames.AppManifest] || {};
 
+      // For SPFx manifest
+      this.envInfoV3.config.isLocalDebug = true;
```

Comment:
Any side effect if setting this for non-spfx project?

Reply:
May be it is safe, because the old local debug always set `config.isLocalDebug = true`
https://github.com/OfficeDev/TeamsFx/blob/179da259aa0f23923390a8eb7438341f0b35fc4b/packages/fx-core/src/core/FxCore.ts#L207